utils/jwtUtil.js

```diff
@@ -91,7 +91,9 @@ function getTokenDetailsFromTokenString(s) {
     if (s) {
       jwt.verify(s, env.tokenSecret, {}, (err, decodedData) => {
         if (err !== null || !decodedData) {
-          return reject(err);
+          return reject(new apiErrors.ForbiddenError({
```

Comment:
@iamigo I added the forbidden status code, to be consistent with the other return.